www/js/config/dynamic_config.js

```diff
@@ -164,23 +164,93 @@ angular.module('emission.config.dynamic', ['emission.plugin.logger'])
         }
     }
 
+    /*
+     * We want to support both old style and new style tokens.
```

Comment:
So old-style tokens have 3 parts, and new style tokens have 4 parts?

Reply:
Yes. Because there are tokens out there (e.g. to smart commute, pueblo, etc) that are in old format.
Once all the old style programs are done, we can remove that backwards compatibility